src/user/formats/hal.ts

```diff
@@ -40,6 +39,13 @@ export function item(user: User, privileges: PrivilegeMap) {
     type: TypeMap.get(user.type),
     privileges
   };
+
+  if (user.type === 3) {
```

Comment:
Is user.type ever tbe literal 3? I think it needs to be compared with ‘UserType.group‘